templates/plugins.local.php.j2

```diff
@@ -1,9 +1,8 @@
-
 <?php
 #{{ ansible_managed }}
 
 /**
  * This file configures the default states of available plugins. All settings in
  * the plugins.*.php files will override those here.
  */
-$plugins['ldap'] = 0;
+$plugins['ldap'] = 1;
```

Comment:
soo the default authentication is `plain` but ldap plugin is needed?

Reply:
pls rereview